### ISSUE TITLE
Versioning

### ISSUE THREAD

**Issue by ksmithut** [shotgunrules/rules#2]
Since this will be an evolving definition, we should set up versioning guidelines, like:
- Does everyone using shotgun have to be aware of the same version or do groups of people subscribe to ranges (semver carat and tilde)?
- Are we declaratively stating that whatever the latest version is is law?
- How do we go about notifying people of upcoming versions?

**Comment by jamesblack** [comment]
We should have an effective startdate for any new rules or new versions, that is no sooner than 1 month from the introduction date, at which point that rule/version will become the new standard.

**Comment by dncrews** [comment]
Agreed. I also think that if we create a section of "House Rules", any given driver should be able to identify which addendums they agree with. Then we should sell stickers. So they can proclaim the rules they support.

**Comment by ksmithut** [comment]
I like the bumper stickers idea. Like eslintrc extends

**Comment by dncrews** [comment]
`Car implements Shotgunrules.org`
